themes/tobac_v1/Example_Updraft_Tracking/Example_Updraft_Tracking.ipynb

```diff
@@ -0,0 +1,7754 @@
+    "Perform feature detection based on midlevel maximum vertical velocity and a range of threshold values."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Determine temporal and spatial sampling of the input data:\n",
+    "dxy,dt=tobac.utils.get_spacings(W_mid_max)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Keyword arguments for feature detection step:\n",
+    "parameters_features={}\n",
+    "parameters_features['position_threshold']='weighted_diff'\n",
+    "parameters_features['sigma_threshold']=0.5\n",
+    "parameters_features['min_num']=3\n",
+    "parameters_features['min_distance']=0\n",
+    "parameters_features['sigma_threshold']=1\n",
+    "parameters_features['threshold']=[3,5,10] #m/s\n",
+    "parameters_features['n_erosion_threshold']=0\n",
+    "parameters_features['n_min_threshold']=3"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "start feature detection based on midlevel column maximum vertical velocity\n",
+      "feature detection performed start saving features\n",
+      "features saved\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Perform feature detection and save results:\n",
+    "print('start feature detection based on midlevel column maximum vertical velocity')\n",
+    "dxy,dt=tobac.utils.get_spacings(W_mid_max)\n",
+    "Features=tobac.themes.tobac_v1.feature_detection_multithreshold(W_mid_max,dxy,**parameters_features)\n",
+    "print('feature detection performed start saving features')\n",
+    "Features.to_hdf(os.path.join(savedir,'Features.h5'),'table')\n",
+    "print('features saved')\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Segmentation:**  \n",
+    "Perform segmentation based on 3D total condensate field to determine cloud volumes associated to identified features:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "parameters_segmentation_TWC={}\n",
+    "parameters_segmentation_TWC['method']='watershed'\n",
+    "parameters_segmentation_TWC['threshold']=0.1e-3  # kg/kg mixing ratio"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Start segmentation based on total water content\n",
+      "segmentation TWC performed, start saving results to files\n",
+      "segmentation TWC performed and saved\n"
+     ]
+    }
+   ],
+   "source": [
+    "print('Start segmentation based on total water content')\n",
+    "Mask_TWC,Features_TWC=tobac.themes.tobac_v1.segmentation_3D(Features,TWC,dxy,**parameters_segmentation_TWC)\n",
```

Comment:
This line break with an `RuntimeError: NetCDF: HDF error` when running with the newest tobac verison `1.3.dev22+ga3cbfb8`. It might be caused by changes in the underlying tobac verison - it needs to be checked again.

Reply:
The issue is resolved, see https://github.com/climate-processes/tobac/pull/36